### ISSUE TITLE
Stabilize configuration format?

### ISSUE THREAD

**Issue by liamdawson** [jwilm/alacritty#1520]
In the README, it states:
> Precompiled binaries will eventually be made available on supported platforms. _This is minimally blocked on a stable config format._
(emphasis mine).
What do we need to achieve to reach that point?

I can see a few issues thus far that probably relate:
* Can't use YAML references (https://github.com/jwilm/alacritty/issues/1127)
* Suggestion to support TOML (https://github.com/jwilm/alacritty/issues/1217)

**Comment by chrisduerr** [comment]
The issue with the configuration file is not that the technology and libraries it uses are unstable, but that the Alacritty configuration is not set in stone. So it's possible and likely that the configuration file will still undergo major changes.

However I do not think that this should hold us back from publishing github releases at this point. I think the bigger problem with precompiled binaries is that there is no version tagging. Which is already tracked in #1240.

I'm going to close this issue since there are no direct steps to take here. If the goal is to have precompiled binaries then #80 and #1240 should be followed. I suspect that once #1240 is closed that #80 will not take much longer to solve.

If you have any further questions, feel free to ask.
